src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -1957,6 +1989,41 @@ public partial struct Single : System.IComparable, System.IComparable<float>, Sy
         public static bool TryParse(string s, System.Globalization.NumberStyles style, System.IFormatProvider provider, out float result) { throw null; }
         public static bool TryParse(string s, out float result) { throw null; }
     }
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
+    public partial struct Span<T>
+    {
+        public static Span<T> Empty { get { throw null; } }
+        public Span(T[] array) { throw null; }
+        public Span(T[] array, int start) { throw null; }
+        public Span(T[] array, int start, int length) { throw null; }
+        [CLSCompliant(false)]
```

Comment:
Can you please explain the difference between [CLSCompliant(false)] and [System.CLSCompliantAttribute(false)] that I see elsewhere (such as https://github.com/dotnet/corefx/blob/master/src/System.Threading.ThreadPool/ref/System.Threading.ThreadPool.cs#L34)?